src/evict/evict_lru.c

```diff
@@ -2516,6 +2532,15 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WTI_EVICT_QUEUE *queue, u_int max_en
 
         page->evict_pass_gen = __wt_atomic_load64(&evict->evict_pass_gen);
 
+        if (__wt_page_is_modified(page))
+            ++pages_seen_dirty;
+        else if (page->modify != NULL)
+            ++pages_seen_clean_with_mod;
```

Comment:
I feel it is better to combine `clean page with mod` and `updates` as one and treat them as `updates`.